airbyte-db/lib/src/main/java/io/airbyte/db/factory/FlywayFactory.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.factory;
+
+import javax.sql.DataSource;
+import org.flywaydb.core.Flyway;
+
+/**
+ * Temporary factory class that provides convenience methods for creating a {@link Flyway}
+ * instances. This class will be removed once the project has been converted to leverage an
+ * application framework to manage the creation and injection of {@link Flyway} objects.
+ *
+ * This class replaces direct calls to {@link io.airbyte.db.Databases}.
```

Comment:
> This class replaces direct calls to {@link io.airbyte.db.Databases}.

Is this true? I looked at the Databases class and I didn't see any mention of Flyaway

Reply:
It's tangentially true.  While the the `Databases` class doesn't create `Flyway` objects, it is currently used to create the `Database` object from which a `DataSource` instance is retrieved to configure the `Flyway` object.  See https://github.com/airbytehq/airbyte/blob/master/airbyte-db/lib/src/main/java/io/airbyte/db/instance/FlywayDatabaseMigrator.java#L60 for details.